cvat/apps/engine/backup.py

```diff
@@ -0,0 +1,550 @@
+            'group',
+            'source',
+            'attributes',
+            'shapes',
+        }
+
+        def _update_attribute(attribute, label):
+            if 'name' in attribute:
+                source, dest = attribute.pop('name'), 'spec_id'
+            else:
+                source, dest = attribute.pop('spec_id'), 'name'
+            attribute[dest] = label_mapping[label]['attributes'][source]
+
+        def _update_label(shape):
+            if 'label_id' in shape:
+                source, dest = shape.pop('label_id'), 'label'
+            elif 'label' in shape:
+                source, dest = shape.pop('label'), 'label_id'
+            shape[dest] = label_mapping[source]['value']
+
+            return source
+
+        for tag in annotations['tags']:
+            label = _update_label(tag)
+            for attr in tag['attributes']:
+                _update_attribute(attr, label)
+            self._prepare_meta(allowed_fields, tag)
+
+        for shape in annotations['shapes']:
+            label = _update_label(shape)
+            for attr in shape['attributes']:
+                _update_attribute(attr, label)
+            self._prepare_meta(allowed_fields, shape)
+
+        for track in annotations['tracks']:
+            label = _update_label(track)
+            for shape in track['shapes']:
+                for attr in shape['attributes']:
+                    _update_attribute(attr, label)
+                self._prepare_meta(allowed_fields, shape)
+
+            for attr in track['attributes']:
+                _update_attribute(attr, label)
+            self._prepare_meta(allowed_fields, track)
+
+        return annotations
+
+    def _prepare_review_meta(self, review):
+        allowed_fields = {
+            'estimated_quality',
+            'status',
+            'issues',
+        }
+        return self._prepare_meta(allowed_fields, review)
+
+    def _prepare_issue_meta(self, issue):
+        allowed_fields = {
+            'frame',
+            'position',
+            'created_date',
+            'resolved_date',
+            'comments',
+        }
+        return self._prepare_meta(allowed_fields, issue)
+
+    def _prepare_comment_meta(self, comment):
+        allowed_fields = {
+            'message',
+            'created_date',
+            'updated_date',
+        }
+        return self._prepare_meta(allowed_fields, comment)
+
+    def _get_db_jobs(self):
+        if self._db_task:
+            db_segments = list(self._db_task.segment_set.all().prefetch_related('job_set'))
+            db_segments.sort(key=lambda i: i.job_set.first().id)
+            db_jobs = (s.job_set.first() for s in db_segments)
+            return db_jobs
+        return ()
+
+class TaskExporter(_TaskBackupBase):
+    def __init__(self, pk, version=Version.V1):
+        self._db_task = models.Task.objects.prefetch_related('data__images').select_related('data__video').get(pk=pk)
+        self._db_data = self._db_task.data
```

Comment:
@azhavoro 
Probably there is the same problem like in the recent fix.
select_related(data__..) breaks prefetch_related(data__...)